src/cli/cli.cpp

```diff
@@ -2439,7 +2383,7 @@ otError Interpreter::ProcessLinkMetricsMgmt(uint8_t aArgsLength, Arg aArgs[])
 
         if (!clear)
         {
-            VerifyOrExit(aArgsLength == 5);
+            VerifyOrExit(!aArgs[4].IsEmpty(), error = OT_ERROR_INVALID_ARGS);
```

Comment:
Shall we ensure that the argument count is exactly as required?
```suggestion
            VerifyOrExit(!aArgs[4].IsEmpty() && aArgs[5].IsEmpty(), error = OT_ERROR_INVALID_ARGS);
```

Reply:
Changed in new push.